src/pages/dar_application/ResearcherInfo.js

```diff
@@ -175,13 +174,14 @@ export default function ResearcherInfo(props) {
             isRendered: (showValidationMessages && !nihValid && !checkCollaborator), className: 'col-lg-12 col-md-12 col-sm-6 col-xs-12 cancel-color required-field-error-span'
           }, ['NIH eRA Authentication is required']),
 
-          div({ className: 'row no-margin' }, [
+          div({ className: 'row no-margin', isRendered: !isLoading }, [
             eRACommons({
               className: 'col-lg-6 col-md-6 col-sm-6 col-xs-12 rp-group',
               destination: eRACommonsDestination,
               onNihStatusUpdate: onNihStatusUpdate,
               location: location,
-              validationError: showValidationMessages
+              validationError: showValidationMessages,
+              currentUser: currentUser
```

Comment:
In a previous comment, I suggested that you get the current user from `Storage.getCurrentUser()` instead of passing it in as a prop. That would let you revert this and have a single place to get the current user as we know about it.

Reply:
We could do that, but then we would need to make an API call to get the User with properties because  Storage.getCurrentUser() does not contain user properties and we need the authorization and expiration props

Reply:
Then let's make a ticket to store the full user in Storage.setCurrentUser so we have what we need to have. The workarounds we're doing are what's making this a more difficult task.